plane/test_fit_plane.py

```diff
@@ -157,6 +157,35 @@ def test_image_to_physical_output_size(self):
         self.assertAlmostEqual(random_image[10,10,1],mapped_image[10,10,1])
         self.assertAlmostEqual(random_image[50,50,2],mapped_image[50,50,2])
 
+    def test_image_physical(self):
```

Comment:
Please rename test function:
test_image_physical --> test_image_to_physical_translations_uv

And rename function below
test_image_to_physical_translations --> test_image_to_physical_translations_xy